parser-typechecker/src/Unison/Codebase/Serialization/V1.hs

```diff
@@ -183,6 +184,11 @@ getText = do
   bs <- getBytes len
   pure $ decodeUtf8 bs
 
+skipText :: MonadGet m => m ()
+skipText = do
```

Comment:
I'd use: https://hackage.haskell.org/package/bytes-0.15.3/docs/Data-Bytes-Get.html#v:skip Seems more likely to have optimized implementation.

Even with the benefit of laziness, I think you'll still be allocating a thunk with the code that's written here.